### PR TITLE
Add context menu "new building block" to IC and PV BBs #1015

### PR DESCRIPTION
**Pull request by rwmcintosh**
Re-using the standard "AddBuildingBlocks" dialog to add either Parameter Values or Initial Conditions building blocks when a user right clicks on the folder node.

Also, double click expands/contracts the folder node similar to other collector/folder nodes

### CODE REVIEW

File: src/MoBi.UI/Views/AddBuildingBlocksToModuleView.cs [hunk 1]
```diff
@@ -32,13 +44,46 @@ protected override void StartValueCheckChanged(bool enabled, LayoutControlItem n
...
+      }
+
+      private void adjustHeight(int heightAdjustment)
+      {
+         Size = new Size(Width, Height + heightAdjustment);
+      }
+
+      private void hideAllItemsExcept(List<BaseLayoutItem> itemsToShow)
+      {
+         var initialGroupHeight = createBuildingBlocksGroup.Size.Height;
```
Comment (msevestre) [inline src/MoBi.UI/Views/AddBuildingBlocksToModuleView.cs:73]: what's with this height here? I don't quite get it

**Review by msevestre — COMMENTED** [review]
I am not sure I get the initial height stuff. 
Also to check on different monitors. Manual resizing =>always issues

File: src/MoBi.UI/Views/AddBuildingBlocksToModuleView.cs (hunk 1 above)
Comment (rwmcintosh) [reply src/MoBi.UI/Views/AddBuildingBlocksToModuleView.cs:73]: That's the height of the layout group before hiding many items.

Later we take the difference and then apply that difference to the dialog height to adjust.

**Comment by rwmcintosh** [comment]
I can't try different physical monitors, but adjusting the resolution and scaling on my monitor indicates that this works at different scales and resolutions.

**Pull request merged**